go.mod

```diff
@@ -33,8 +33,8 @@ require (
 	github.com/spf13/cobra v0.0.3
 	github.com/stretchr/testify v1.8.1
 	golang.org/x/crypto v0.1.0
-	golang.org/x/sys v0.1.0
-	golang.org/x/text v0.4.0
+	golang.org/x/sys v0.7.0
+	golang.org/x/text v0.9.0
```

Comment:
Spent an hour reviewing dependency changes, didn't find anything alarming. Below are some notes...

Reviewed dependency change:
* golang.org/x/text - no functional changes aside from updating dependencies. [v0.4.0 was released Oct 14 2022, reviewed 8 commits.](https://github.com/golang/text/commits/master)
* golang.org/x/sys - [v0.1.0 was released Oct 11 2022, reviewed 65 commits](https://github.com/golang/sys/commits/master?before=39c2d6a01da63fd9e091bbd231476308d3b193d2+35&branch=master&qualified_name=refs%2Fheads%2Fmaster).
    * [there are some glibc updates in the native code for this package](https://github.com/golang/sys/commit/7a75290698de16ec0a9d0581ac45fa3808fa9431)
    * lots and lots of small changes, hard to say if any of it is used by our software. Most is gated to platforms we don't use. Some general things related to Ioctl that I don't think we use (maybe used for ledger wallet support?)
* golang.org/x/net - [v0.1.0 was released Oct 19 2022, skimmed 30-40 commits]. Minor bugfixes and perf improvements to http server code, lots of minor/unrelated html handling/parsing changes

Running out of steam. I haven't looked at:
* **golang.org/x/mod**
* **golang.org/x/tools**
* **golang.org/x/term**

Reply:
Please note that `go.mod` has partially reverted since I simplified some of the functionality.